app/src/main/java/com/github/se/cyrcle/ui/review/AllReviewsScreen.kt

```diff
@@ -352,12 +362,28 @@ fun ReviewCard(
                     modifier = Modifier.fillMaxWidth(),
                     horizontalArrangement = Arrangement.SpaceBetween,
                     verticalAlignment = Alignment.CenterVertically) {
-                      androidx.compose.material3.Text(
-                          text = title,
-                          style = MaterialTheme.typography.bodySmall,
-                          modifier = Modifier.weight(1f).testTag("ReviewTitle$index"),
-                          maxLines = 1,
-                          overflow = TextOverflow.Ellipsis)
+                      if (range.color == "rainbow") {
+                          androidx.compose.material3.Text(
+                              text = "[${range.symbol}$level] $title",
+                              style = MaterialTheme.typography.bodySmall,
+                              modifier = Modifier
+                                  .weight(1f)
+                                  .testTag("ReviewTitle$index"),
+                              maxLines = 1,
+                              overflow = TextOverflow.Ellipsis
+                          )
+                      } else {
+                          androidx.compose.material3.Text(
+                              text = "[${range.symbol}$level] $title",
```

Comment:
Please here note that the card is also used in the "ViewProfileScreen", where the title is NOT the owner's username. The title is the name of the parking

Reply:
Oh thanks for pointing this out ! I didnt realise that the ViewProfileScreen was also calling ReviewCard but with the parking name as argument for title instead of the owner's username. So it was adding a tag to parking name... not what we want

Reply:
No worries. The fix you pushed is perfect. This is on the author of the PR that introduced this card in another screen. They should have moved it to molecules to make it clear. (Yes I am speaking about myself in 3rd person)